src/main/webapp/js/maps-page-loader.js

```diff
@@ -87,7 +87,7 @@ async function createCinemaMarkerForDisplay(map, lat, lng, cinemaName, key, upda
  * This function fetches messages for the given parent key.
  */
 async function getMessagesForKey(key, containerDiv) {
```

Comment:
Can this be made synchronous so we don't have to use the await?

Reply:
same for the createCinemaMarkerForDisplay function above?

Reply:
fetch methods are by default asyncronous